import sys


def errors(code: int, display=False):
    """
    Функция предназначена для отображения описания возникшей ошибки и завершения работы приложения

    :param code: Код ошибки, соответствующий индексу списка ответов, тип integer
    :param display: Указывает выводить описание ошибки в консоль или нет, тип bool
    :return: Описание ошибки отображается на экране, и приложение закрывается
    """
    answer = ["Передан код завершения сеанса",
              "Не правильно указан порт",
              "Не правильно указан адрес",
              "Ошибка доступа к порту. Порт уже занят, либо у вас недостаточно прав на его использование",
              "Сервер не отвечает",
              "Сервер принудительно разорвал соединение",
              "Сервер разорвал установленное подключение",
              "Ошибка установки соединения",
              ]
    try:
        if display:
            print(answer[code])
    except IndexError:
        return 1
    sys.exit()
